### PR TITLE
Add back link

### PR DESCRIPTION
**Pull request by deberny**
During user testing we have discovered a user need to go back to previous page.

Added back links at the end of raise repair pages. This gives the
user the ability to go back and edit their input.

Added improvements to booking call back time logic. We have decided that for picking a call back slots it's better if don't use "avoid school run ". Because in this scenario it's more likely that a person can answer the phone during that time. And this removes complexity.

Improve the wording to add more clarity.

### CODE REVIEW

File: app/views/raiserepair/book-appointment-contact.html [hunk 1]
```diff
@@ -30,13 +30,17 @@ <h2 class="heading-medium">
         <input class="form-control" id="telephone-number" type="number" name="telephone-number" required>
       </div>
 
-      <p>We will send you a reminder before your appointment.<p>
+      <p>We will send you a text reminder before your repair appointment.<p>
 
         <div class="form-group">
-          <input class="button" type="submit" value="Continue">
+          <input class="button" type="submit" value="Confirm and book a repair">
```
Comment (dgmstuart) [inline app/views/raiserepair/book-appointment-contact.html]: I'm not sure about the phrasing "book a repair"

Maybe "Confirm and book this appointment" makes more sense?

File: app/views/raiserepair/describe-repair-address.html [hunk 2]
```diff
@@ -56,12 +56,17 @@ <h2 class="heading-medium">
...
+      <a href="/raiserepair/describe-repair-contact/">Back to add contact details</a>
+    </p>
+
     </div>
 
     <div class="column-one-third">
       <aside role="submited-steps">
         <h3 class="heading-large"><span class="heading-secondary">Your report</span></h3>
         <p class="heading-small">Problem</p>
+        <p>Something else</p>
```
Comment (dgmstuart) [inline app/views/raiserepair/describe-repair-address.html]: Might this be confusing to the resident? Do we need it?

File: app/views/raiserepair/describe-repair-address.html [hunk 3]
```diff
@@ -74,16 +79,17 @@ <h3 class="heading-large"><span class="heading-secondary">Your report</span></h3
         <p>Tel number: {{ data['telephone-number'] }}</p>
         <p>Available for call:
 
-          {% if data['morningCall'] %}
```
Comment (dgmstuart) [inline app/views/raiserepair/describe-repair-address.html:77]: As discussed, let's remove "school run"

File: app/views/raiserepair/repair-details.html [hunk 4]
```diff
@@ -55,7 +55,19 @@ <h2 class="heading-medium">
         </div>
 
       </form>
+      <p>
+        <a href="/raiserepair/what-is-problem/">Back to what do you need to be fixed</a>
```
Comment (dgmstuart) [inline app/views/raiserepair/repair-details.html]: Might 'what do you need to be fixed?' make more sense in quotes?

**Review by dgmstuart — CHANGES_REQUESTED** [review]

File: app/views/raiserepair/describe-repair-address.html (hunk 3 above)
Comment (deberny) [reply app/views/raiserepair/describe-repair-address.html:77]: done 👍

**Review by dgmstuart — APPROVED** [review]
Latest changes look good - will merge

**Pull request merged**